src/pages/BuyDomains/BuyDomains.module.scss

```diff
@@ -0,0 +1,68 @@
+@use 'zero-ui/src/styles/dimensions';
+@use 'zero-ui/src/styles/colors';
+@use 'zero-ui/src/styles/typography';
+
+.BuyDomains {
+	&__Page_Container {
```

Comment:
I commented similar in `BuyDomains.tsx` - you don't need the `&__Page_Container` if you're using modules. Instead, just use the class `.PageContainer`.